### PR TITLE
Swap to Github Actions

### PR DESCRIPTION
**Pull request by kkellyy**
# Problem 
Travis CI is going away soon and it doesn't seem to be properly releasing new versions to npm anyway

# Solution
Crib [this template](https://github.com/kkellyy/dataloader-codegen/new/master?filename=.github%2Fworkflows%2Fnpm-publish.yml&workflow_template=npm-publish) + [a small bit of Tron's setup for python](https://github.com/Yelp/Tron/blob/master/.github/workflows/ci.yml)

# Verification
[The test job is working on my fork](https://github.com/kkellyy/dataloader-codegen/runs/2605562970?check_suite_focus=true).

I believe I'll have to test the release job "live"  though. It seems straightforward from the examples, but I believe we'll need to add our npm upload token [here](https://github.com/Yelp/dataloader-codegen/settings/secrets/actions/new).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ minimal: node_modules build
 
 venv: Makefile requirements-dev.txt
 	rm -rf venv
-	virtualenv venv --python=$(PYTHON3)
+	$(PYTHON3) -m venv venv
```
Comment (magicmark) [inline Makefile:8]: nice! til

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        needs: test
+        runs-on: ubuntu-20.04
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+              with:
+                  node-version: 12
+                  registry-url: https://registry.npmjs.org/
+            - run: yarn
+            - run: npm publish
```
Comment (magicmark) [inline .github/workflows/release.yml:30]: per https://docs.github.com/en/actions/guides/publishing-nodejs-packages#publishing-packages-using-yarn

```suggestion
            - run: yarn publish
```

**Review by magicmark — COMMENTED** [review]
nice! can we also update https://github.com/Yelp/dataloader-codegen/blob/master/PUBLISH.md to reflect this?

**Comment by kkellyy** [comment]
> nice! can we also update https://github.com/Yelp/dataloader-codegen/blob/master/PUBLISH.md to reflect this?

I think the only change will be the example URL which I don't have a good link to yet :X. I was thinking of doing that in an separate PR or just directly on master once I've successfully released `v0.3.0`

**Review by magicmark — APPROVED** [review]
thanks!!

**Pull request merged**